openstudiocore/src/model/PhotovoltaicPerformanceSimple.hpp

```diff
@@ -0,0 +1,111 @@
+/**********************************************************************
+ *  Copyright (c) 2008-2015, Alliance for Sustainable Energy.
+ *  All rights reserved.
+ *
+ *  This library is free software; you can redistribute it and/or
+ *  modify it under the terms of the GNU Lesser General Public
+ *  License as published by the Free Software Foundation; either
+ *  version 2.1 of the License, or (at your option) any later version.
+ *
+ *  This library is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *  Lesser General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Lesser General Public
+ *  License along with this library; if not, write to the Free Software
+ *  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ **********************************************************************/
+
+#ifndef MODEL_PHOTOVOLTAICPERFORMANCESIMPLE_HPP
+#define MODEL_PHOTOVOLTAICPERFORMANCESIMPLE_HPP
+
+#include <model/ModelAPI.hpp>
+#include "PhotovoltaicPerformance.hpp"
+
+namespace openstudio {
+
+namespace model {
+
+class Schedule;
+
+namespace detail {
+
+  class PhotovoltaicPerformanceSimple_Impl;
+
+} // detail
+
+/** PhotovoltaicPerformanceSimple is a PhotovoltaicPerformance that wraps the OpenStudio IDD object 'OS:PhotovoltaicPerformance:Simple'. */
+class MODEL_API PhotovoltaicPerformanceSimple : public PhotovoltaicPerformance {
+ public:
+  /** @name Constructors and Destructors */
+  //@{
+
+  explicit PhotovoltaicPerformanceSimple(const Model& model);
```

Comment:
@evanweaver this object doesn't make a lot of sense on its own, it is designed to be owned by a generatorphotovoltaic object.  i had originally made this a protected ctor to avoid weirdness about ownership when assigning to a generatorphotovoltaic.  This is one place we should chat with @kbenne and @mbadams5